### DISCUSSION TITLE
RT-AX53U support?

### DISCUSSION THREAD

**Discussion by LovanDog** [Vaskivskyi/asusrouter#296] [Q&A] [answered]
didnt find any information about RT-AX53U on support list. what about this model?

**Comment by Vaskivskyi** [comment]
Hello. The model should be supported. For now, no one reported usage of AsusRouter with RT-AX53U, but the available FW, as well as HW, is fully compatible and should not have any unexpected issues

**Comment by Vaskivskyi** [comment, marked as answer]
I will close the discussion, since `RT-AX53U` has been on the supported list already for some time. Just cleaning up to make sure, no discussions are left unanswered.
